tests/adhoc/nullness/enabled/positive.next.bsl

```diff
@@ -1,12 +1,24 @@
 
-tests\adhoc\nullness\enabled\positive.fs(8,24): warning FS3271: The 'nullness checking' language feature is not enabled. This use of a nullness checking construct will be ignored.
+tests\adhoc\nullness\enabled\positive.fs(4,21): error FS0010: Unexpected symbol '|' in binding. Expected '=' or other token.
 
-tests\adhoc\nullness\enabled\positive.fs(51,15): warning FS3271: The 'nullness checking' language feature is not enabled. This use of a nullness checking construct will be ignored.
+tests\adhoc\nullness\enabled\positive.fs(5,21): error FS0010: Unexpected symbol '|' in binding. Expected '=' or other token.
 
-tests\adhoc\nullness\enabled\positive.fs(56,19): warning FS3271: The 'nullness checking' language feature is not enabled. This use of a nullness checking construct will be ignored.
+tests\adhoc\nullness\enabled\positive.fs(12,31): error FS0010: Unexpected symbol '|' in expression
 
-tests\adhoc\nullness\enabled\positive.fs(63,19): warning FS3271: The 'nullness checking' language feature is not enabled. This use of a nullness checking construct will be ignored.
+tests\adhoc\nullness\enabled\positive.fs(18,28): error FS0010: Unexpected symbol '|' in expression
 
-tests\adhoc\nullness\enabled\positive.fs(193,17): error FS0001: The type 'List<int>' does not have 'null' as a proper value
+tests\adhoc\nullness\enabled\positive.fs(26,15): error FS0010: Unexpected symbol '|' in binding. Expected '=' or other token.
 
-tests\adhoc\nullness\enabled\positive.fs(195,17): error FS0001: The type 'List<String>' does not have 'null' as a proper value
+tests\adhoc\nullness\enabled\positive.fs(35,24): error FS0010: Unexpected symbol '|' in binding. Expected '=' or other token.
```

Comment:
Some extra errors here, are they expected?

Reply:
Looks like they're associated with return types?

Reply:
This baseline is for F# language version preceeding the nullness feature.

I did the parser changes incl. the LexFilter change under a preview flag, so that F# for older versions behaves as if this never existed in order to eliminate breaking changes.